.github/workflows/librariangen.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
This will be covered by the regular kokro CI already as we run tests for all modules. If you think this needs it own action I would purpose we use path filtering: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#example-including-paths

Reply:
I'm okay with that - but I'm slightly surprised that we aren't seeing Kokoro failures already, given the missing files that are added (well, moved) in this PR. Happy for us to look into that separately though - I may be looking in the wrong place.

Removed this file in an extra comment.